documentation/modules/mtv-selected-packages-2-7.adoc

```diff
@@ -142,6 +153,7 @@ The following listed packages are from the `virt-v2v` guest conversion pod:
 |nbdkit-curl-plugin-1.38.3-1.el9.x86_64
 |nbdkit-curl-plugin-1.38.3-1.el9.x86_64
 |nbdkit-curl-plugin-1.38.3-1.el9.x86_64
+|nbdkit-curl-plugin-1.38.3-1.el9.x86_64
 
 |NBD proxy / forward plugin for nbdkit
```

Comment:
Pedantic: IBM says not to use spaces around a forward slash. Also to write out the words "and", "of" or "or", unless this is an industry standard.